code/modules/ruins/lavaland_anomalies.dm

```diff
@@ -0,0 +1,80 @@
+//Black Box
+
+/obj/machinery/smartfridge/black_box
+	name = "black box"
+	desc = "A completely indestructible chunk of crystal, rumoured to predate the start of this universe. It looks like you could store things inside it."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_on = "blackbox"
+	icon_off = "blackbox"
+	luminosity = 8
+	max_n_of_items = 10
+	pixel_y = -4
+	use_power = 0
+	var/duplicate = FALSE
+	var/memory_saved = FALSE
+	var/list/stored_items = list()
+
```

Comment:
Requires a blacklist of what can be inserted!
You dont want someone storing the nuke disk in it!